Assets/HoloToolkit/Input/Scripts/OnGazeEnterEvent.cs

```diff
@@ -1,21 +0,0 @@
-﻿using UnityEngine;
```

Comment:
Please keep those files but just make them use the new design pattern.
"The desire to let designers use the Inspector to set up Events this way is great for doing quick prototyping, but isn't necessarily the best practice in a production environment" this is your opinion, mine is the exact opposite hence why I added those files ;-)
So I guess as long as they don't rely on SendMessage anymore there's no reason to remove them.
Thanks!